packages/ui/src/lib/modal/Modal.svelte

```diff
@@ -9,19 +9,19 @@ interface Props {
   name?: string;
   top?: boolean;
   ignoreFocusOut?: boolean;
+  overflowVisible?: boolean;
   onclose?: () => void;
-  children?: Snippet;
 }
 
 let modal: HTMLDivElement;
 let {
   name = 'drop-down-dialog',
   top = false,
   ignoreFocusOut = false,
+  overflowVisible = false,
```

Comment:
I feel like having something like `overflow?: 'auto' | 'visible'` is more resilient: if we need to add change, we can just add a value to the allowed type

Reply:
FWIW I would still remove entirely and keep the API contract/code simple.